modules/livestore/live_store.go

```diff
@@ -325,6 +327,8 @@ func (s *LiveStore) stopping(error) error {
 		}
 	}
 
+	s.stopAllBackgroundProcesses()
```

Comment:
should we call `s.completeQueues.Stop()`? 

looking at the code I actually don't know if it would do anything which is odd, but it seems appropriate.

Reply:
It's called inside that function 😄

Reply:
i'm losing my mind! thx mario 🙏